### PR TITLE
Delay Trino broadcast by a week

### PR DESCRIPTION
**Pull request by bitsondatadev**
Manfred and I didn't realize the schedule was still for a week after last weeks broadcast. Will push it back a week.

### CODE REVIEW

File: broadcast/index.md [hunk 1]
```diff
@@ -45,10 +45,9 @@ See us live every two weeks!
...
 |---|---|---|
-| Thursday, Jan 14, 2021 <br> at 1:00 pm -0500 | Distributed hash-join  |   |
-| Thursday, Jan 28, 2021 <br> at 1:00 pm -0500 |Dynamic filtering   | Software Engineer and Starburst Co-Founder, [Karol Sobczak](https://twitter.com/sopel39) and Software Engineer, Raunaq Morarka  |
+| Thursday, Jan 21, 2021 <br> at 1:00 pm -0500 | Distributed hash-join  |   |
+| Thursday, Feb 04, 2021 <br> at 1:00 pm -0500 |Dynamic filtering   | Software Engineer and Starburst Co-Founder, [Karol Sobczak](https://twitter.com/sopel39) and Software Engineer, Raunaq Morarka  |
 
 </div>
 </div>
 </div>
-</div>
```
Comment (martint) [inline broadcast/index.md:54]: Unrelated?

Comment (bitsondatadev) [reply broadcast/index.md:54]: yeah...it was extra

Comment (bitsondatadev) [reply broadcast/index.md:54]: added it back
